### ISSUE TITLE
cmd_verify lstrip output

### ISSUE THREAD

**Issue by dainok** [ktbyers/netmiko#2707]
I discovered that cmd_verify=True strip left part of output (lstrip).

Output from the router:

~~~
R1#show vrf
  Name                             Default RD            Protocols   Interfaces
  int                              <not set>                         Lo0
~~~

Netmiko with cmd_verify:

~~~
>>> net_connect.send_command("show vrf")
'Name                             Default RD            Protocols   Interfaces\n  int                              <not set>                         Lo0'
>>> net_connect.send_command("show vrf", use_textfsm=True)
'Name                             Default RD            Protocols   Interfaces\n  int                              <not set>                         Lo0'
~~~

Netmiko without cmd_verify:

~~~
>>> net_connect.send_command("show vrf", cmd_verify=False)
'  Name                             Default RD            Protocols   Interfaces\n  int                              <not set>                         Lo0'
>>> net_connect.send_command("show vrf", cmd_verify=False, use_textfsm=True)
[{'name': 'int', 'default_rd': '<not set>', 'protocols': 'Lo0', 'interfaces': []}]
~~~

You can see that the double initial spaces are stripped with cmd_verify, and this affect TextFSM parsing.

**Comment by ktbyers** [comment]
Which platform is this?

Is this your own template or ntc-templates?

**Comment by dainok** [comment]
platform: cisco_ios
netmiko: 3.4.0

I'm using NTC templates, but my point is that Netmiko is altering the output from the device. Don't know if it's by design or not.

**Comment by ktbyers** [comment]
Can you re-test it with Netmiko 4.0.0? It might be fixed with that.

**Comment by dainok** [comment]
Fixed in 4.0.0, thank you.

~~~
>>> net_connect.send_command("show vrf")
'  Name                             Default RD            Protocols   Interfaces\n  int                              <not set>                         Lo0'
~~~

**Comment by ktbyers** [comment]
Okay, great.
